lib/usdUfe/ufe/Utils.h

```diff
@@ -394,6 +394,25 @@ bool isEditTargetLayerModifiable(
     const PXR_NS::UsdStageWeakPtr stage,
     std::string*                  errMsg = nullptr);
 
+//! Check if a vector of paths could be added/removed on a given relationship.
+//! \param relationship The relationship to be edited.
+//! \param targetsToAdd The optional paths to be added to the relationship. If
+//!        not nullptr, the entries will be adjusted to only keep the paths that
+//!        are allowed to be added to the relationship.
+//! \param targetsToRemove The optional paths to be removed from the
+//!        relationship. If not nullptr, the entries will be adjusted to only
+//!        keep the paths that are allowed to be removed from the relationship.
+//! \param errMsg The optional error message to be filled in case of any target
+//!        not being able to be added/removed.
+//! \return True, if one ore more of the targets given are allowed to be
+//!        added/removed to the relationship in the stage's local Layer Stack.
```

Comment:
Not sure if we should require that *all* desired changes be allowed? Was that specified in the story that we allow if a single one is correct and all the other incorrect?